### PR TITLE
Remove waiter test

### PR DESCRIPTION
**Pull request by taniarezhytska**

### CODE REVIEW

File: TAF_EasyRest/TestFramework/Pages/Owner/OwnerPanelRestaurantsPage.cs [hunk 1]
```diff
@@ -1,34 +1,37 @@
...
         return this;
     }
 
-
-
-
+    public override void GoToUrl()
+    {
+        driver.Navigate().GoToUrl(baseUrl + _pageUrl);
+        Thread.Sleep(3000);
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Pages/Owner/OwnerPanelRestaurantsPage.cs:35]: Make a comment in lines like this to change them later.

File: TAF_EasyRest/TestFramework/Test/OwnerTests.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
+﻿namespace TestFramework.Test
+{
+    [TestFixture]
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Test/OwnerTests.cs:3]: No need. Works without [TestFixture]

File: TAF_EasyRest/TestFramework/Test/OwnerTests.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        {
+            SignInPage signInPage = new SignInPage(ChromeDriver);
+            string email = "jasonbrown@test.com";
+            string password = "1111";
+
+            signInPage.GoToUrl();
+            signInPage.SendKeysToEmailField(email)
+                .SendKeysToPasswordField(password)
+                .ClickSignInButton();
+            Thread.Sleep(3000);
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Test/OwnerTests.cs:25]: Make a comment to change Thread.Sleep later.

**Review by LoveMyDaughter — APPROVED** [review]
Approve. Please make sure to make changes in the next pull-request corresponding to my comments.

File: TAF_EasyRest/TestFramework/PageComponents/Owner/ManageRestaurantPageComponent.cs [hunk 4]
```diff
@@ -20,10 +20,11 @@ public ManageMenuesPageComponent ClickMenuesButton()
             _menuesButton.Click();
             return new ManageMenuesPageComponent(driver);
         }
-        public ManageRestaurantPageComponent ClickWaiterssButton()
+        public ManageWaitersPage ClickWaitersButton()
         {
             _waitersButton.Click();
-            return this;
+            Thread.Sleep(1000);
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/PageComponents/Owner/ManageRestaurantPageComponent.cs:26]: please add comment to change this later

File: TAF_EasyRest/TestFramework/PageComponents/Owner/OwnerPanelPageComponent.cs [hunk 5]
```diff
@@ -13,13 +13,14 @@ public OwnerPanelPageComponent(IWebDriver driver)
         private IWebElement _manageButton => driver.FindElement(By.XPath("//span[text() = 'Manage']/ancestor::a"));
         private IWebElement _archiveButton => driver.FindElement(By.XPath("//span[text() = 'Archive']/ancestor::li"));
 
-        public OwnerEditRestaurantPage ClickSignInButton()
+        public OwnerEditRestaurantPage ClickManageButton()
         {
             _manageButton.Click();
+            Thread.Sleep(1000);
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/PageComponents/Owner/OwnerPanelPageComponent.cs:19]: please add a comment to change this later

File: TAF_EasyRest/TestFramework/PageComponents/Owner/WaiterItemPageComponent.cs [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+        {
+            _waiterElement = element;
+        }
+
+        private IWebElement _removeButton => _waiterElement.FindElement(By.TagName("button"));
+
+        public void ClickRemoveButton()
+        {
+            _removeButton.Click();
+            Thread.Sleep(1000);
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/PageComponents/Owner/WaiterItemPageComponent.cs:16]: please add a comment to change this later

File: TAF_EasyRest/TestFramework/Pages/Owner/ManageWaitersPage.cs [hunk 7]
```diff
@@ -10,23 +10,22 @@ public class ManageWaitersPage : BasePage
         public ManageRestaurantPageComponent ManageRestaurantPageComponent { get; }
         public UserMenuHeaderButtonPageComponent UserMenuHeaderButtonPageComponent { get; }
 
+        public List<WaiterItemPageComponent> WaiterItems
+        {
+            get { return _items.Select(i => new WaiterItemPageComponent(i)).ToList(); }
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/Pages/Owner/ManageWaitersPage.cs:15]: very nice!

**Review by dmatsev — APPROVED** [review]
Approve with comments

**Pull request merged**